### PR TITLE
Index file modified and functions added

### PR DESCRIPTION
**Pull request by rishab9750**
## Code Modification

Improved the server index file and added the functional invocation.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -18,20 +18,34 @@ app.use('/api/docs', swaggerUI.serve, swaggerUI.setup(swaggerDocument))
...
+    });
+}
 
-app.get('/', (req,res) =>{
-    res.send('Express backend running')
-})
-app.get('/api', (req, res) => {
-    res.send({message: `Welcome to &lt;LinkShortener&gt; API`});
-});
+async function init() {
```
Comment (cherylli) [inline server/index.js:32]: does this need to be async?

**Review by cherylli — CHANGES_REQUESTED** [review]
functions order are weird. YOu have the init in the middle of all the other functions and you made an initAppServer() but it's not in init()

**Comment by rishab9750** [comment]
> functions order are weird. YOu have the init in the middle of all the other functions and you made an initAppServer() but it's not in init() Could you also use arrow functions please?

initAppServer() is not inside init() as it is being initiated after a successful database connection.

File: server/index.js (hunk 1 above)
Comment (rishab9750) [reply server/index.js:32]: No this async keyword can be removed.

**Comment by cherylli** [comment]
Sorry, I see it now, but why do we need a new function for a 2 line code that we are not reusing?

**Comment by rishab9750** [comment]
> 

I agree we can move it inside of the "then" statement and it would work as expected, what I did just improves the readability of the code and makes it clear that after database connectivity we are going to start the server.

**Comment by cherylli** [comment]
everything looks good but i'd prefer not having the initAppServer function, since it's a really short non reusable function

**Pull request closed without merging**